test/suite/test_checkpoint_snapshot02.py

```diff
@@ -63,6 +63,7 @@ def conn_config(self):
         return config
 
     def moresetup(self):
+        self.ignoreStdoutPatternIfExists('Eviction took more than 1 minute') # FIXME-WT-15478
```

Comment:
isnt it test_checkpoint_snapshot03?

Reply:
Good catch :) thanks!
Unfortunately it didn't (and doesn't) fail for me so I'm not actually able to check if the fix works.
I scheduled these tasks on evg for presubmit check but it's not guaranteed to fail even if the bug is still there.